packages/i18n/locales/cs/translation.json

```diff
@@ -0,0 +1,629 @@
+      "daodaoLogo": "DAO DAO Logo",
+      "daodaoWithVersion": "DAO DAO v{{version}}",
+      "daosLogo": "DAO's Logo",
+      "depositFromStargazeQuestion": "Chcete vložit NFT ze Stargaze?",
+      "depositNftsModalSubtitle": "Vyberte NFT které chcete vložit do {{daoName}} ",
+      "disabled": "Vypnuto",
+      "disconnectWalletTooltip": "Odpojit peněženku",
+      "draftSavedAtTime": "Návrh byl uložen v {{time}}.",
+      "draftSaving": "Ukládání návrhu...",
+      "draftStillSaved": "Návrh bude v bezpečí.",
+      "easilyShareLink": "Jednoduše zkopírujte odkaz dole.",
+      "enabled": "Povoleno",
+      "enterNameBeforeSavingDraft": "Enter a name before saving a draft",
+      "establishedAbbr": "Přibližně",
+      "executeSmartContractActionDescription": "Potvrďte smlouvu smart kontraktu",
+      "failing": "Chyba",
+      "fetchedAtTime": "Vyhledáno {{time}}",
+      "getTokensOn": "Získejte tokeny na {{name}}.",
+      "goToParent": "Přejít do hlavního DAO",
+      "govTokenAddress": "Hlasovací token",
+      "groupAddress": "CW4 skupina",
+      "historySinceDate": "Historie od {{date}}",
+      "inboxDescription": "Otevřít návrh",
+      "instantiateSmartContractActionDescription": "Potvrdit smart kontrakt",
+      "jsonIsValid": "JSON je správný",
+      "keplrModalWalletExplanation": "Vaše peněženka je vaše digitální identita v blockchainu. Její vlastnictví vám umožní komunikovat s web3 aplikacemi, jako je DAO DAO.",
+      "loading": "Načítání...",
+      "logo": "Logo",
+      "majority": "Většina",
+      "manageMembersActionDescription": "Přidání, aktualizace nebo odebrání členů.",
+      "manageSubDaosActionDescription": "Rozpoznat nebo odebrat SubDAO.",
+      "members": "Členi",
+      "membershipDaoNotMemberInfo_dao": "Nejste členem {{daoName}}. Chcete-li se stát členem, musí stávající členové hlasovat pro vaše přidání.",
+      "membershipDaoNotMemberInfo_proposal": "Nebyl jste členem {{daoName}} když byl tento návrh vytvořen, takže o něm nemůžete hlasovat.",
+      "migrateSmartContractActionDescription": "Migrace smlouvy CosmWasm na nové ID.",
+      "mintActionDescription": "Vytvořit nové governance tokeny.",
+      "name": "Název",
+      "newMemberExplanation": "Pro přidání nového člena musí DAO schválit návrh.",
+      "nftImage": "NFT obrázek",
+      "no": "Ne",
+      "noDaosPinnedYet": "Nemáte žádné připnuté DAO",
+      "noDisplayName": "Zatím jste nepřidali jméno",
+      "noNftsYet": "Nemáte žádné NFT které by se dalo ocenit.",
+      "noProposalsYet": "Zatím nebyly předloženy žádné návrhy, o kterých by se hlasovalo.",
+      "noSubDaosYet": "Zatím neexistuje žádné SubDAO",
+      "noVote": "Ne",
+      "none": "Nic",
+      "notAdmin": "Vaše DAO není správcem této smlouvy. V takovém případě nebude návrh proveditelný.",
+      "notMet": "Nesplněno",
+      "nothingFound": "Nic nebylo nalezeno.",
+      "numNftsSelected_one": "{{count}} vybrané NFT",
+      "numNftsSelected_other": "{{count}} vybraných NFTs",
```

Comment:
dal bych jenom NFT bez toho s. udělal bych to tak všude

Reply:
Ne, chtějí tam množný tvar, nad tím je jednotný